lib/config/configuration.go

```diff
@@ -181,10 +181,14 @@ type CommandLineFlags struct {
 	OpenSSHConfigPath string
 	// OpenSSHKeysPath is the path to write teleport keys and certs into
 	OpenSSHKeysPath string
+	// OpenSSHKeysBackupPath is the path to write teleport keys and certs into after rotation
+	OpenSSHKeysBackupPath string
 	// AdditionalPrincipals are a list of extra principals to include when generating host keys.
 	AdditionalPrincipals string
 	// RestartOpenSSH indicates whether openssh should be restarted or not.
 	RestartOpenSSH bool
+	// RotateOpenSSH indicates that the OpenSSHCA certs should be rotated
```

Comment:
Why is this up to the command line arguments and not up to the state of the cluster? The host ssh certificate (that's the SSH half of the host Node credentials) to use depends on the state of the cluster's `host` `cert_authority`, and since we're going to be merely consumers of the `openssh` `cert_authority`, all we need to do there is fetch it - because we have to do that anyway to know if it has changed - and then update the list of trusted user cert authorities for opensshd with all the ssh keys we find in there. I don't see how the user should be allowed to specify which "kind" of rotation we should do.